ansible/group_vars/all.yml

```diff
@@ -143,3 +143,12 @@ aws_region: None
 # this reads "'s3.{{ aws_region }}.amazonaws.com' if aws_region else None"
 aws_endpoint: '{{ aws_region and "s3." + aws_region + ".amazonaws.com" }}'
 aws_versioning_enabled: true
+
+airflow_secret_key: '{{ secrets.AIRFLOW_SECRET_KEY | default(None) }}'
+airflow_flask_key: '{{ secrets.AIRFLOW_FLASK_KEY | default(None) }}'
```

Comment:
This will end up as a string `'None'`

Reply:
That seems to be what all the other credentials in this file do. Happy to go a different route if there is a preferable one, but as a default that seems at worst harmless.